core/src/main/scripts/importer/validateData.py

```diff
@@ -2138,11 +2145,43 @@ class FusionValidator(Validator):
         'Method',
         'Frame']
     REQUIRE_COLUMN_ORDER = True
+    ALLOW_BLANKS = True
+
+    def __init__(self, *args, ** kwargs):
+        super(FusionValidator, self).__init__(*args, **kwargs)
+        self.fusion_lines = {}
 
     def checkLine(self, data):
+
         super(FusionValidator, self).checkLine(data)
-        # TODO check the values
 
+        # parse hugo and entrez to validate them together
```

Comment:
this section seems identical to what we have for MutationsExtendedValidator. Please remove this code duplication